GEOSaana_GridComp/GSI_GridComp/etc/gsi.rc.tmpl

```diff
@@ -172,9 +172,9 @@ OBS_INPUT::
    iasibufr       iasi        metop-c     iasi_metop-c          0.0     3      0        ncep_mtiasi_bufr
    atmsbufr       atms        npp         atms_npp              0.0     1      0        ncep_atms_bufr
    atmsbufr       atms        n20         atms_n20              0.0     1      0        ncep_atms_bufr
-   atmsbufr       atms        n21         atms_n21              0.0     1      0        ncep_atms_bufr
```

Comment:
delete the atms_n21 line?

Reply:
I guess my base line  does not have that atms_n21 line.  I will add that line back.